openzwavemqtt/models/node.py

```diff
@@ -199,6 +199,17 @@ def neighbors(self) -> List[int]:
         """Return Neighbors."""
         return self.data.get("Neighbors")
 
+    @property
+    def values(self):
+        """Return list of OZWValue child items."""
+        _values = []
+        for instance in self.collections["instance"]:
+            for cc in instance.collections["commandclass"]:
```

Comment:
We should make automatic helpers to iterate over collections. We can do that in a future PR.